src/fixtures/stubbed_logstash_index_pattern.js

```diff
@@ -41,13 +46,9 @@ export default function stubbedLogstashIndexPatternService() {
     };
   });
 
-  const indexPattern = new StubIndexPattern(
-    'logstash-*',
-    (cfg) => cfg,
-    'time',
-    fields,
-    npSetup.core
-  );
+  const indexPattern = new StubIndexPattern('logstash-*', (cfg) => cfg, 'time', fields, {
+    uiSettings: uiSettingSetupMock,
+  });
 
```

Comment:
This is the only actual code change of the PR. The `StubIndexPattern` is only accessing uiSettings API, so I created an inline mock to replace the `'../legacy/ui/public/new_platform/new_platform.karma_mock'` import.